### PR TITLE
Homepage bottom info

### PR DESCRIPTION
**Pull request by maaverik**

### CODE REVIEW

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 1]
```diff
@@ -41,17 +41,27 @@ export const HowToJoin = () => (
 
 export const ExtraInfo = () => (
   <div className={styles['extra-info']}>
-    <div>
+    <div className={styles.card}>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Home/StaticInfos.jsx:44]: Can you try `styles['card']`?

File: simplq/src/styles/homePage.module.scss [hunk 2]
```diff
@@ -112,15 +112,33 @@
...
+    width: 20%;
+    height: 20%;
+  }
+
+  .brief-description {
+    font-weight: bold;
+    color: white;
+  }
+
+  .detailed-descriptions {
```
Comment (daltonfury42) [inline simplq/src/styles/homePage.module.scss:139]: detailed-description

File: simplq/src/styles/homePage.module.scss [hunk 3]
```diff
@@ -112,15 +112,33 @@
...
+  }
+
+  img {
+    width: 20%;
+    height: 20%;
+  }
+
+  .brief-description {
+    font-weight: bold;
+    color: white;
```
Comment (daltonfury42) [inline simplq/src/styles/homePage.module.scss:136]: I think we should do a `text-transform: capitalize` here. It's not in design, but looks better.

**Review by daltonfury42 — APPROVED** [review]

**Comment by maaverik** [comment]
All changes done.

**Pull request merged**